components/thumb-controls/index.js

```diff
@@ -213,13 +221,25 @@ AFRAME.registerComponent('thumb-controls-debug', {
     if (!data.enabled && !AFRAME.utils.getUrlParameter('debug-thumb')) { return; }
     console.log('%c debug-thumb', 'background: #111; color: red');
 
-    // Stub.
-    el.components['tracked-controls'].handleAxes = () => {};
+    var GetTrackedControlsProperties = function () {
+      // Stub.
+      el.components['tracked-controls'].handleAxes = () => {};
+
+      axis = [0, 0, 0];
+      axisMoveEventDetail = {axis: axis};
+      el.components['tracked-controls'].axis = axis;
+      el.components['thumb-controls'].axis = axis;
+    };
 
-    axis = [0, 0, 0];
-    axisMoveEventDetail = {axis: axis};
-    el.components['tracked-controls'].axis = axis;
-    el.components['thumb-controls'].axis = axis;
+    // There may exist a tracked-controls when this component is initialized
+    if (el.components['tracked-controls']) {
+      GetTrackedControlsProperties();
```

Comment:
In JS convention is to start function names with lower case and upper case names are reserved for constructors